packages/playground/src/pages/pull-request-changes-page.tsx

```diff
@@ -3,7 +3,10 @@ import PlaygroundPullRequestChangesSettings from '../settings/pull-request-chang
 import { SkeletonList } from '../components/loaders/skeleton-list'
 import { NoData } from '../components/no-data'
 import { ListActions, Spacer, SplitButton } from '@harnessio/canary'
+import * as data from '../data/mockDiffViewerdata'
+
 import PullRequestChanges from '../components/pull-request/pull-request-changes'
+type K = 'a' | 'b' | 'c' | 'd' | 'e' | 'j'
```

Comment:
What exactly are the letters? `a`, `b`,...

Reply:
removed it was to pick the different mock diff viewer data